internal/flix/flix.go

```diff
@@ -0,0 +1,142 @@
+package flix
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/onflow/flixkit-go"
+
+	"github.com/onflow/flow-cli/flowkit/output"
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/internal/scripts"
+	"github.com/onflow/flow-cli/internal/transactions"
+
+	"github.com/onflow/flow-cli/flowkit"
+
+	"github.com/spf13/cobra"
+)
+
+type flixFlags struct {
+	ArgsJSON    string   `default:"" flag:"args-json" info:"arguments in JSON-Cadence format"`
+	BlockID     string   `default:"" flag:"block-id" info:"block ID to execute the script at"`
+	BlockHeight uint64   `default:"" flag:"block-height" info:"block height to execute the script at"`
+	Signer      string   `default:"" flag:"signer" info:"Account name from configuration used to sign the transaction as proposer, payer and suthorizer"`
+	Proposer    string   `default:"" flag:"proposer" info:"Account name from configuration used as proposer"`
+	Payer       string   `default:"" flag:"payer" info:"Account name from configuration used as payer"`
+	Authorizers []string `default:"" flag:"authorizer" info:"Name of a single or multiple comma-separated accounts used as authorizers from configuration"`
+	Include     []string `default:"" flag:"include" info:"Fields to include in the output"`
+	Exclude     []string `default:"" flag:"exclude" info:"Fields to exclude from the output (events)"`
+	GasLimit    uint64   `default:"1000" flag:"gas-limit" info:"transaction gas limit"`
+}
+
+var flags = flixFlags{}
+
+var FlixCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "flix <id | name | path>",
+		Short:   "Execute flix with given id, name or path",
+		Example: "flow flix multiply 2 3 --network testnet",
```

Comment:
NIT If we don't need to provide a network let's not so it's not complicating example.
```suggestion
		Example: "flow flix multiply 2 3",
```